apollo-router-benchmarks/benches/basic_composition.rs

```diff
@@ -1,6 +1,7 @@
 use apollo_router::ApolloRouter;
 use apollo_router_core::{
-    FetchError, Fetcher, PreparedQuery, Request, Response, Router, ServiceRegistry,
+    ByteString, FetchError, Fetcher, PreparedQuery, Request, Response, Router, ServiceRegistry,
```

Comment:
I don't think it makes sense to reexport types from serde_json_bytes in apollo-router-core. Maybe the benchmark could import serde_json_bytes instead?